packages/patternfly-4/react-core/src/components/Modal/ModalContent.js

```diff
@@ -33,7 +35,9 @@ const propTypes = {
   /** Creates a small version of the Modal */
   isSmall: PropTypes.bool,
   /** id to use for Modal Box description */
-  id: PropTypes.string.isRequired,
```

Comment:
\Removing id will break the API.  Perhaps keep it and update the description for now.   We can can open an issue about changing the name in resolve that when we are ready for our next major release.

Reply:
@tlabaj 
I'm happy to change it back, but (just to make sure I understand) ModalContent is a component that is loaded by Modal.js. I assumed that since the Modal component is what the users would import, that this was more of an internal API. The API on the generated site doesn't change. But this is still a breaking change?

Reply:
changing the name of the variable back does not change anything in the code outside of Modal.js [line #109](https://github.com/patternfly/patternfly-react/blob/d6e7c30eaa26d2b9cbb236c58660997cc9b87950/packages/patternfly-4/react-core/src/components/Modal/Modal.js#L109) and the references to the property inside of ModalContent so even though it's a less descriptive name, it's not an inaccurate name, so it may not need to be changed at all if it's really considered a breaking change.

Reply:
You are right since it is internal, this would not break a consumer API.  And yes the new name is more descriptive so I agree it should change.